### PR TITLE
Bastion host

### PR DESCRIPTION
**Pull request by sevos**

### CODE REVIEW

File: lib/baustelle/cloud_formation/bastion_host.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            'zone' => zone_name = config.fetch('dns_zone'),
+            'hostname' => "bastion-#{stack_name.underscore.gsub('_', '-')}"
+          }
+        }.to_yaml
+
+        iam_role = parent_iam_role.inherit("bastion_host", {
+                                             'zone_update' => {
+                                               'resource' => "arn:aws:route53:::hostedzone/#{get_zone_id(zone_name)}",
+                                               'action' => 'route53:ChangeResourceRecordSets'
+                                             }
```
Comment (suxor42) [inline lib/baustelle/cloud_formation/bastion_host.rb]: why does the host need this right?

File: spec/baustelle/stack_template_spec.rb [hunk 2]
```diff
@@ -443,6 +451,55 @@ def group_option_settings(option_settings)
         expect(template[:Resources]['ApplicationOnlyStagingEnvProd']).to be_nil
         expect(template[:Resources]['ApplicationOnlyStagingEnvLoadTest']).to be_nil
       end
+
+      context 'generates bastion host configuration' do
+        it 'security group' do
+          expect_resource template, "BastionSecurityGroup",
+                          of_type: 'AWS::EC2::SecurityGroup'
```
Comment (suxor42) [inline spec/baustelle/stack_template_spec.rb]: maybe check that SSH access is allowed from at least somewhere?!

File: lib/baustelle/cloud_formation/bastion_host.rb (hunk 1 above)
Comment (sevos) [inline lib/baustelle/cloud_formation/bastion_host.rb]: after spinning up in ASG it needs to set the DNS

Comment (sevos) [inline lib/baustelle/cloud_formation/bastion_host.rb]: to point to itself, you can't use CFN for route53 w/o ELB

**Pull request merged**